diff --git a/src/svd/CH32V003.svd b/src/svd/CH32V003.svd
index b439064..d1891cd 100644
--- a/src/svd/CH32V003.svd
+++ b/src/svd/CH32V003.svd
@@ -9231,5 +9231,30 @@ xmlns:xs="http://www.w3.org/2001/XMLSchema-instance" xs:noNamespaceSchemaLocatio
         </register>           
 		</registers>
 	</peripheral>
+
+    <peripheral>
+      <name>StubInterrupts_GeneratorIgnoreMe</name>
+      <interrupt>
+        <name>NMI</name>
+        <description>Non-maskable interrupt</description>
+        <value>2</value>
+      </interrupt>
+      <interrupt>
+        <name>HardFault</name>
+        <description>Abnormal interruptions</description>
+        <value>3</value>
+      </interrupt>
+      <interrupt>
+        <name>SysTick</name>
+        <description>System timer interrupt</description>
+        <value>12</value>
+      </interrupt>
+      <interrupt>
+        <name>SW</name>
+        <description>Software interrupt</description>
+        <value>14</value>
+      </interrupt>
+    </peripheral>
+
 	</peripherals>
  </device>
